pynblint/nb_linting.py

```diff
@@ -262,6 +262,20 @@ def cells_too_long(notebook: Notebook) -> List[Cell]:
     ]
 
 
+def detect_multiline_comment(notebook: Notebook) -> List[Cell]:
```

Comment:
```suggestion
def long_multiline_python_comment(notebook: Notebook) -> List[Cell]:
```

I'd suggest changing the name of this function for consistency with the others. We never use the word _detect_, as it is implicit in all cases.